sale_order_product_recommendation_packaging_default/wizards/sale_order_recommendation.py

```diff
@@ -26,6 +27,7 @@ class SaleOrderRecommendationLine(models.TransientModel):
         digits="Product Unit of Measure",
         help="Quantity of packagings to sell.",
     )
+    require_packaging = fields.Boolean(compute="_compute_require_packaging")
```

Comment:
issue: you can use a readonly related field instead, which is essentially the same:

```suggestion
    require_packaging = fields.Boolean(related="product_id.require_packaging", readonly=True)
```